readme.md

```diff
@@ -63,6 +65,7 @@ All methods can be chained together.
 | callFunction | ``cb`` Callback, ``thisArg`` this Object to bind to the callback function | Call a callback function. The first parameter to the callback ``elements`` which contains all DOM nodes used in the typewriter effect. |
 | changeDeleteSpeed | ``speed`` Number or 'natural' | The speed at which to delete the characters, lower number is faster. |
 | changeDelay | ``delay`` Number or 'natural' | Change the delay when typing out each character |
+| clearWrapper | - | Clear the text wrapper immediately, from outside the event loop (not affected by `loop`). |
```

Comment:
I would recommend leaving this method in even though it is a little out of place - it is used to clear the text wrapper after the initial prefill without adding to the event loop, and it seems to make sense to document it in case there are other use cases.